geometry_msgs/Point[] points
uint8 size
geometry_msgs/Point location
uint8 identity
